airbyte-integrations/connectors/source-monday/source_monday/schemas/updates.json

```diff
@@ -8,7 +8,7 @@
         "type": ["null", "object"],
         "additionalProperties": true,
         "properties": {
-          "id": { "type": ["null", "integer"] }
+          "id": { "type": ["null", "integer", "string"] }
```

Comment:
Can normalization be applied to multiple types of data?

Reply:
this particular change was enough to make CAT tests passing, to me it seemed that this was enough, but I'll check again if we should leave only "string" type here